bin/webpack.js

```diff
@@ -1,19 +1,57 @@
 #!/usr/bin/env node
 
+const cp = require("child_process");
+const inquirer = require("inquirer");
+
+function runCommand(command) {
+	cp.exec(command, (error, stdout, stderr) => {
+		if(!error) {
+			console.log("webpack-cli installed successfully");
+			return true;
```

Comment:
returning a value doesn't make sense here. It's an async function.

Maybe you want to wrap this in a Promise:

``` js
return new Promise((resolve, reject) => {
  ...
  resolve(true);
  ...
});